gammapy/cube/tests/test_sherpa_.py

```diff
@@ -55,3 +59,138 @@ def test_sherpa_crab_fit():
                  2.240989]
 
     assert_allclose(result.parvals, reference, rtol=1E-5)
+
+
+@requires_dependency('sherpa')
+@requires_data('gammapy-extra')
+def testCombinedModel3DInt():
+    from sherpa.models import PowLaw1D, TableModel
+    from sherpa.estmethods import Covariance
+    from sherpa.optmethods import NelderMead
+    from sherpa.stats import Cash
+    from sherpa.fit import Fit
+    from ..sherpa_ import CombinedModel3DInt
+
+    # Set the counts
+    filename = gammapy_extra.filename('test_datasets/cube/counts_cube.fits')
+    counts_3d = SkyCube.read(filename)
+    cube = counts_3d.to_sherpa_data3d(dstype='Data3DInt')
+
+    # Set the bkg
+    filename = gammapy_extra.filename('test_datasets/cube/bkg_cube.fits')
+    bkg_3d = SkyCube.read(filename)
+    bkg = TableModel('bkg')
+    bkg.load(None, bkg_3d.data.value.ravel())
+    bkg.ampl = 1
+    bkg.ampl.freeze()
+
+    # Set the exposure
+    filename = gammapy_extra.filename('test_datasets/cube/exposure_cube.fits')
+    exposure_3d = SkyCube.read(filename)
+    i_nan = np.where(np.isnan(exposure_3d.data))
+    exposure_3d.data[i_nan] = 0
+    # In order to have the exposure in cm2 s
+    exposure_3d.data = exposure_3d.data * 1e4
+
+    # Set the mean psf model
+    filename = gammapy_extra.filename('test_datasets/cube/psf_cube.fits')
+    psf_3d = SkyCube.read(filename)
+
+    # Setup combined spatial and spectral model
+    spatial_model = NormGauss2DInt('spatial-model')
+    spectral_model = PowLaw1D('spectral-model')
+    source_model = CombinedModel3DInt(use_psf=True, exposure=exposure_3d, psf=psf_3d,
+                                      spatial_model=spatial_model, spectral_model=spectral_model)
+
+    # Set starting values
+    center = SkyCoord.from_name("Crab").galactic
```

Comment:
Can you rather set the position manually, instead of querying a web database using `from_name()`. Otherwise this will fail, if users don't have internet connection (and might cause random fails on Travis-CI...)